src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.managed.cs

```diff
@@ -652,7 +652,7 @@ private void UpdateAverageLineHeight()
 
 		private double CalculatePanelMeasureBreadth() => _materializedLines.Select(l => GetDesiredBreadth(l.FirstView)).MaxOrDefault()
 #if __WASM__
-			+ GetBreadth(XamlParent.ScrollViewer.ScrollBarSize)
+			+ GetBreadth(XamlParent?.ScrollViewer.ScrollBarSize ?? default)
```

Comment:
I'm curious why you need the `?? default` here.  Isn't what the compiler gives by default?

Reply:
nope :) `default` is the non-nullable value type (`default(Size)`) in this case, it would be `Size?` otherwise.

simple example:
![image](https://user-images.githubusercontent.com/8270914/126680780-8abdf3cc-04bf-4abd-bf5c-639cda891d0b.png)